flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaInternalProducerITCase.java

```diff
@@ -102,8 +106,9 @@ public void testHappyPath() throws Exception {
         deleteTestTopic(topicName);
     }
 
-    @Test(timeout = 30000L)
-    public void testResumeTransaction() throws Exception {
+    @Test
+    @Timeout(30L)
```

Comment:
time unit should be specified

Reply:
By default it takes seconds, do we still need to specify?